actions/workflows/test_ip_address.yaml

```diff
@@ -82,5 +96,7 @@ tasks:
         do:
           - noop
       - when: "{{ succeeded() and (ctx().num_results != 1 or ctx().record_state == 'assigned' or ctx().record_claimed == true) }}"
+        publish:
```

Comment:
Maybe break this up into two different errors.

- One error for claimed (meaning no DNS data is associated with it, but the address is reserved)
- One error for already assigned, and tell us what value is currently assigned